content/source/docs/enterprise/api/state-versions.html.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: enterprise2
+page_title: "State Versions - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-state-versions"
+---
+
+# State Versions API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## Create a State Version
+
+`POST /workspaces/:workspace_id/state-versions`
+
+| Parameter       | Description                                               |
+| --------------- | --------------------------------------------------------- |
+| `:workspace_id` | The workspace ID to create the new state version. |
+
+Creates a state version and sets it as the current state version for the given
+workspace.
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) or [team token](../users-teams-organizations/service-accounts.html#team-service-accounts).
+
+Status  | Response                                     | Reason
+--------|----------------------------------------------|----------
+[201][] | [JSON API document][]                        | Successfully created a state version
+[404][] | [JSON API error object][]                    | Workspace not found, or user unauthorized to perform action
+[409][] | [JSON API error object][]                    | Conflict; check the error object for more information
+[412][] | [JSON API error object][]                    | Precondition failed; check the error object for more information
+[422][] | [JSON API error object][]                    | Malformed request body (missing attributes, wrong types, etc.)
+
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+
+Key path                    | Type    | Default   | Description
+----------------------------|---------|-----------|------------
+`data.type`                 | string  |           | Must be `"state-versions"`.
+`data.attributes.serial`    | integer |           | The serial of the state version
+`data.attributes.md5`       | string  |           | An MD5 hash of the raw state version
+`data.attributes.lineage`   | string  | (nothing) | Lineage of the state version
```

Comment:
What does default of nothing mean here?

Reply:
Just that it isn't required. Earlier versions of Terraform did not have lineage.

https://github.com/hashicorp/terraform-website/blob/master/content/source/docs/enterprise/api/_template.md#request-body

Reply:
I wonder if it's worth clarifying that, and maybe what purpose the lineage serves? I don't think we'd want people to create new states without lineage, right?